transit_odp/timetables/extract.py

```diff
@@ -95,15 +97,19 @@ def extract(self) -> ExtractedData:
         # Extract JourneyPatternSections, TimingLinks and RouteLinks
         logger.debug("Extracting journey_patterns_sections")
         jp_sections, timing_links = self.extract_journey_pattern_sections()
+
         logger.debug("Finished extracting journey_patterns_sections")
 
         vehicle_journeys = pd.DataFrame()
+        flexible_vehicle_journeys = pd.DataFrame()
         serviced_organisations = pd.DataFrame()
         operating_profiles = pd.DataFrame()
         if is_timetable_visualiser_active:
             # Extract VehicleJourneys
             logger.debug("Extracting vehicle_journeys")
             vehicle_journeys = self.extract_vehicle_journeys()
+            logger.debug("Extracting flexible vehicle journeys")
+            flexible_vehicle_journeys = self.extract_flexible_vehicle_journeys()
```

Comment:
Any reason the flexible vehicle journeys are extracted? This is part of the dqs changes

Reply:
will remove these changes